src/classes/audioContextWrapper.tsx

```diff
@@ -10,12 +10,15 @@ class AudioContextWrapper {
     masterGainNode : GainNode
     filterNode : BiquadFilterNode
     waveshaperNode : WaveshaperNodeWrapper
-    oscilators: OscillatorWrapper[]
+    primaryOscillators: OscillatorWrapper[]
+    secondaryOscillators: OscillatorWrapper[]
```

Comment:
This implementation doesn't scale, as we'd need to do a rewrite it we wanted to add a third oscillator